motion-1/Overview.bs

```diff
@@ -92,68 +92,140 @@ In addition to the property-specific values listed in their definitions, all pro
 
 Values have the following meanings:
 
-<dl dfn-for="motion-path">
-  <dt><<basic-shape>></dt>
+<dl dfn-for="offset-path">
+  <dt><<angle>></dt>
   <dd>
-    The motion path is a basic shape as specified in CSS Shapes [[!CSS-SHAPES]]. The initial position and the initial direction for basic shapes are defined as follows:
+    The path is a closed line segment that starts from the current position of the element and has the degree of the angle as specified in <<angle>> from the Y-axis. The path ends at the edge of the containing block.
     <dl>
-      <dt><<circle()>></dt>
-      <dt><<ellipse()>></dt>
-      <dd>The <a>initial position</a> is defined by the point where a virtual tangent to the circle/ellipse would reach the top vertical position. The <a>initial direction</a> is 90 degrees.</dd>
-      <dt><<inset()>></dt>
-      <dd>The <a>initial position</a> is defined by the left top corner of the rectangle. The <a>initial direction</a> is 0 degree.</dd>
-      <dt><<polygon()>></dt>
-      <dd>The <a>initial position</a> is defined by the first coordinate pair of the polygon. The <a>initial direction</a> is defined by the vector connecting the initial position with the next following coordinate pair that isn't equal to the initial position. If there is no such unequal coordinate pair, the initial direction is defined with 0 degree.</dd>
+        <dt><dfn>contain</dfn></dt>
+        <dd>
+           Adjust the value of 'offset-distance' when the element goes out of the path. When overflowing occurs, the distance value is reduced, until the end point of the path is on the edge of the element.
+        </dd>
     </dl>
-  </dd>
-  <dt><dfn>path()</dfn> = path([ <<fill-rule>> ,]? <<string>>)</dt>
-  <dd>
-    <p>The <<string>> represents an SVG Path data string. The path data string must be conform to the grammar and parsing rules of SVG 1.1 [[!SVG11]]. The initial position is defined by the first “move to” argument in the path string. For the initial direction follow SVG 1.1 [[!SVG11]].
-    <p><<fill-rule>> - The filling rule used to determine the interior of the path. See 'fill-rule' property in SVG for details. Possible values are ''nonzero'' or ''evenodd''. Default value when omitted is ''nonzero''.
-  </dd>
-  <dt><<url>></dt>
-  <dd>
-    References an SVG <a>shape element</a> and uses its geometry as motion path. See SVG 1.1 for more information about the initial position and initial direction [[!SVG11]].
-  </dd>
-  <dt dfn-type=value><dfn>none</dfn></dt>
-  <dd>No motion path gets created.</dd>
+
+	<p class="note">Note: Defining a path with <<angle>>, 
+	the element can be positioned in the polar coordinate system.
+	In polar coordinates, a pole is the reference point and points are described 
+	as been a certain distance from it, as also a certain angle from the polar axis.
```

Comment:
I will make the definition for polar coordinates clear as follows: 

> The polar coordinate system is a two-dimensional coordinate system in which each point on a plane
	is determined by a distance from a fixed point and an angle from a fixed line.
	The fixed point which is similar to the origin of a Cartesian coordinate system is called the pole,
	and the fixed line which is a ray from the pole is the polar axis.
	This coordinate system specifies the position of a point with a certain distance from the pole
	and a certain angle measured from the polar axis to the segment connecting the pole.